src/Fixer/LanguageConstruct/ClassKeywordRemoveFixer.php

```diff
@@ -95,6 +95,7 @@ private function storeImports(Tokens $tokens, $startIndex, $endIndex)
         $tokensAnalyzer = new TokensAnalyzer($tokens);
         $this->imports = [];
 
+        /** @var int $index */
         foreach ($tokensAnalyzer->getImportUseIndexes() as $index) {
```

Comment:
Couldnt this method be annoted to return a int instead?

Reply:
Unfortunately no, it has `@return int[]|int[][]`, depending of the boolean [argument](https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v2.16.1/src/Tokenizer/TokensAnalyzer.php#L67).